db/users/procedures.sql

```diff
@@ -0,0 +1,21 @@
+/*Procedures of session*/
+DROP PROCEDURE IF EXISTS save_user;
+DELIMITER $$
+CREATE PROCEDURE save_user(
+    IN p_name VARCHAR(255),
+    IN p_email VARCHAR(255),
+    IN p_password VARCHAR(255),
+    IN p_code VARCHAR(50)
+)
+BEGIN
+    DECLARE status VARCHAR(255);
+    DECLARE user_type INT;
+
+
+    SET status = 'notverified'; 
+    SET user_type = 1;
```

Comment:
you could make an alternate table sql command to change the status and user_type attributes to have a default value instead of insert one manually